README.md

```diff
@@ -162,6 +149,4 @@ I would like to thank Micorverse for the guidances
 
```

Comment:
| 🔴 Broken Account Handles |
| --- |
| ![image](https://github.com/ernestkorir/vet_clinic/assets/93039359/a8090496-c11d-443f-abe4-0bbdda14380a) |

- [ ] Kindly fix your account handles because they are broken right now 😨 KIndly fix them to make your readme descriptive and professional 👍